doc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+openfisca_core==4.2.0.dev3
```

Comment:
Relative path doesn't work well in `requirements.txt`, so the doc must refer to the pip version.